cosmovisor/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 2
+sidebar_position: 3
```

Comment:
```suggestion
sidebar_position: 1
```

Reply:
no, I don't want cosmovisor to be first in the list